packages/next/taskfile-swc.js

```diff
@@ -25,14 +25,15 @@ module.exports = function (task) {
 
       const isClient = serverOrClient === 'client'
 
+      /** @type {import('@swc/core').Options} */
       const swcClientOptions = {
         module: {
           type: 'commonjs',
           ignoreDynamic: true,
         },
         jsc: {
           loose: true,
-
+          externalHelpers: true,
```

Comment:
I wonder if we could use the `next-swc` binary here for transforming instead of installing `@swc/core` as well, this could help us ensure we're matching behavior as this is already enabled there.

Reply:
That's a good idea. It also helps eliminate the inconsistency. Let me see what I can do.

Reply:
@ijjk I have tried to re-use the file `next/build/swc/index.js`, but the file is dependent on the precompiled `@napi-rs/triple`.

Maybe we should do it in another PR?

Reply:
We could potentially expose an env variable to signal it should use the direct module instead of the compiled one e.g. `if (process.env.__NEXT_TASKR) { require('@napi-rs/triple') }` 

Yeah we can definitely investigate this in a separate PR